### PR TITLE
fix operationRef dup

### PR DESCRIPTION
**Pull request by zwwhdls**
### Ⅰ. Describe what this PR does
fix operationRef keep add the same name when using cron datamigrate

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: api/v1alpha1/dataset_types.go [hunk 1]
```diff
@@ -353,7 +353,7 @@ func (dataset *Dataset) SetDataOperationInProgress(operationType string, name st
 		}
 		return
 	}
-	if dataset.Status.OperationRef[operationType] == "" {
+	if dataset.Status.OperationRef[operationType] == "" || strings.Contains(dataset.Status.OperationRef[operationType], name) {
```
Comment (cheyang) [inline api/v1alpha1/dataset_types.go:356]: I think it's better to use `equal` than `contain`?

Comment (TrafalgarZZZ) [inline api/v1alpha1/dataset_types.go:356]: `strings.Contains` may not be accurate when there are multiple cron DataMigrate with similar name. I think we can split the string and check it.

Comment (xliuqq) [inline api/v1alpha1/dataset_types.go:356]: If "A,B" contains "A" , the result should be "A,B" not "A".

Comment (zwwhdls) [reply api/v1alpha1/dataset_types.go]: fixed

Comment (zwwhdls) [reply api/v1alpha1/dataset_types.go]: done

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**